bbn-test-4/finality-providers/scripts/fp-check-tx.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+
+# USAGE:
+# ./fp-check-tx.sh
+
+# check if the self-lock staking transaction has all the correct parameters set.
+
+CWD="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+STAKERCLI_BIN="${STAKERCLI_BIN:-stakercli}"
+SIGNED_TX="${SIGNED_TX}"
+FP_BTC_PK="${FP_BTC_PK}"
+
+if ! command -v $STAKERCLI_BIN &> /dev/null; then
+  echo "⚠️ $STAKERCLI_BIN command could not be found!"
+  echo "Install it by checking https://github.com/babylonchain/btc-staker.git"
```

Comment:
Should we provide the tag that should be used here?

Reply:
https://github.com/babylonchain/networks/pull/193/commits/5de3f63d7cf13897a17b9bd1b6ff2867a3af36a4